packages/fxa-event-broker/lib/sentry.ts

```diff
@@ -0,0 +1,178 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Hapi from '@hapi/hapi';
+import Hoek from '@hapi/hoek';
+import * as Sentry from '@sentry/node';
+import { SQS } from 'aws-sdk';
+import { Consumer } from 'sqs-consumer';
+import { PassThrough } from 'stream';
+
+// Matches uid, session, oauth and other common tokens which we would
+// prefer not to include in Sentry reports.
+const TOKENREGEX = /[a-fA-F0-9]{32,}/gi;
```

Comment:
Do any email addresses make it to the event broker that we should also filter?

Reply:
They do! I added an email regex (yes, I know).